tests/Spec/Nri/Ui/Carousel.elm

```diff
@@ -146,6 +147,15 @@ viewWithPreviousAndNextControlsSpec =
                     |> clickButton "Previous"
                     |> ensureSlideIsVisible "slide-0"
                     |> done
+        , test "does not crash with 0 slides" <|
```

Comment:
When 0 slides are passed, `findPreviousAndNextSlides` breaks with a divide by zero error. Maybe an approach like is used in FocusLoop might work here?